rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -453,7 +471,7 @@ private String executeFunction(BiFunction<RestClient, Set<String>, InputStream>
         return "";
     }
 
-    private String executeFunctionReturnStreamAsString(Function<RestClient, InputStream> function)
+    private String executeFunctionReturnStreamAsStringWithoutNewLines(Function<RestClient, InputStream> function)
             throws ServiceException, RestServerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)